category_theory/limits/cones.lean

```diff
@@ -30,7 +30,7 @@ variables {J C} (F : J ⥤ C)
 natural transformations from the constant functor with value `X` to `F`.
 An object representing this functor is a limit of `F`.
 -/
-def cones : Cᵒᵖ ⥤ Type _ := (const (Jᵒᵖ)) ⋙ (op_inv J C) ⋙ (yoneda.obj F)
+def cones : Cᵒᵖ ⥤ Type v := (const (Jᵒᵖ)) ⋙ (op_inv J C) ⋙ (yoneda.obj F)
```

Comment:
A minor improvement would be to left-associate the parentheses here, so that the "functoriality" stuff below is more uniform between the `cones` and `cocones` cases.

Reply:
Done.